docs/src/content/docs/for-users/create-manage-groups.md

```diff
@@ -28,3 +28,16 @@ To remove a user:
 2. Within your groups, select the name of the group that you'd like to remove a user from
 3. Using the 'user' interface at the bottom of the page, click the red button next to the user you wish to remove
 4. The user is now removed
+
+## Edit group information
+
+Anyone in the group can edit group information such as the group name, contact email and adress.
+
+1. Log into your account and navigate to your account page ('My account' in the top-right of the website)
+2. Within your groups, select the name of the group that you would like to edit
+3. In the group detail view, click the 'Edit group' button at the top
+4. Update any information in the edit group form that you would like to change and then click the 'Update group' button
+
+:::caution
+If you change the name of a group the center name in corresponding ENA submissions will also change.
+:::
```

Comment:
```suggestion
```

Reply:
Very pathoplexus specific at the moment so better to deduplicate and only keep in pathoplexus docs